style.css

```diff
@@ -144,160 +155,91 @@
   text-align: center;
 }
 
-.title1 {
-  text-align: center;
-  font-size: 24px;
-  font-weight: 700;
-}
-
-.Languages {
+.abilites {
   display: flex;
-  flex-direction: row;
-  margin-bottom: 20px;
+  justify-content: center;
   align-items: center;
-  list-style-type: none;
-  color: #172b4d;
-}
-
-.chevron {
-  border: 1px solid #dfe1e6;
-  padding: 10px 10px 5px 10px;
-  width: 40px;
-  border-radius: 45px;
-  margin-left: 45%;
-  margin-bottom: 15px;
+  margin: 16px 0 16px 0;
+  padding: 0 0 0 0;
 }
 
-.Languages .language {
-  color: #3a4a42;
+.a1 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
   font-size: 12px;
-  text-align: center;
+  color: #3a4a42;
   background-color: #ebf0ee;
-  border-radius: 4px;
-  padding: 8px 9px;
-  margin-right: 10px;
-}
-
-.b_div {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  padding: 5px 5px 1px 12px;
-  gap: 10px;
-  margin: 13px 0 24px 0;
+  border-radius: 5px;
+  line-height: 133%;
+  padding: 12px 12px;
+  margin-right: 8px;
+  letter-spacing: 0.03em;
 }
 
-.button {
+.button,
+.get-button {
   background-color: #36b37f;
   font-weight: 600px;
   font-size: 17px;
   color: white;
   border-radius: 4px;
-  width: 120px;
-  height: 47px;
+  width: 122px;
+  height: 48px;
   border: 2px solid #36b37f;
+  transition: background-color 1s;
 }
 
-.button:hover {
-  transform: rotate(360deg);
-  transform: scale(1.01);
-}
-
-.button:active {
+.get-button:active {
   background: #008552;
   border-radius: 4px;
 }
 
-.About-me {
-  background-color: #fff;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-image: url(./Images/illustration-\ about\ me-\ mobile12.png);
-  background-repeat: no-repeat;
-  background-position: center top;
-}
-
-.descr {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-end;
-}
-
-.resume {
-  display: flex;
-  flex-direction: row;
-  justify-content: left;
+.get-button:hover {
+  background-color: #d0d9d4;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
 }
```

Comment:
Please consider adding the functionality of `hover and press` on the button. So when the button is hovered or pressed its style changes as follows. Currently, your styles are not in accordance with the design. To achieve this you can add the following functionality of code into your CSS file.

## Solution:--

```
.classname:hover{
    background-color: #36B37F;
    color: white;
}
```

```
.classname:active{
    background-color: #008552;
    color: white;
}
```

## Demonstration:--

![image](https://user-images.githubusercontent.com/94223367/222924036-ec7c1353-54de-484a-b446-68be07b7ae09.png)